src/basic/login-util.h

```diff
@@ -4,6 +4,13 @@
 #include <stdbool.h>
 #include <unistd.h>
 
+#define SD_LOGIND_ROOT_CHECK_INHIBITORS           (UINT64_C(1) << 0)
+/* For internal use only */
+#define SD_LOGIND_INTERACTIVE                     (UINT64_C(1) << 1)
```

Comment:
If this is only for internal use, then it should `UINT64_C(1) << 63` would seem more natural. If we later add another flag, it'll look strange to have it `<< 2`.